### PR TITLE
Feature/sfi 86 GitHub docker chybajuce info

### PR DESCRIPTION
**Pull request by Luke-1988**

### CODE REVIEW

File: sf-docker/.env [hunk 1]
```diff
@@ -25,9 +25,9 @@ NoSqlDataStorageDisabled=false
 
 # Using template for Gstreamer pipeline
 #jetson
-GstPipelineTemplate=uridecodebin uri={0} source::latency=0 ! queue max-size-buffers=1 leaky=downstream ! nvvidconv ! video/x-raw, format=(string)BGRx ! videoconvert ! video/x-raw, format=(string)BGR ! appsink
+#GstPipelineTemplate=uridecodebin uri={0} source::latency=0 ! queue max-size-buffers=1 leaky=downstream ! nvvidconv ! video/x-raw, format=(string)BGRx ! videoconvert ! video/x-raw, format=(string)BGR ! appsink
 #linux
-#GstPipelineTemplate=uridecodebin uri={0} source::latency=0 ! queue max-size-buffers=1 leaky=downstream ! nvvideoconvert ! video/x-raw, format=(string)BGRx ! videoconvert ! video/x-raw, format=(string)BGR ! appsink
+GstPipelineTemplate=uridecodebin uri={0} source::latency=0 ! queue max-size-buffers=1 leaky=downstream ! nvvideoconvert ! video/x-raw, format=(string)BGRx ! videoconvert ! video/x-raw, format=(string)BGR ! appsink
```
Comment (marek-nereca) [inline sf-docker/.env:30]: jetson pipeline was uncommented on purpose 
but i understand, x64 linux is more common environment

**Review by marek-nereca — APPROVED** [review]

File: sf-docker/.env [hunk 2]
```diff
@@ -25,9 +25,9 @@ NoSqlDataStorageDisabled=false
 
 # Using template for Gstreamer pipeline
 #jetson
-GstPipelineTemplate=uridecodebin uri={0} source::latency=0 ! queue max-size-buffers=1 leaky=downstream ! nvvidconv ! video/x-raw, format=(string)BGRx ! videoconvert ! video/x-raw, format=(string)BGR ! appsink
+#GstPipelineTemplate=uridecodebin uri={0} source::latency=0 ! queue max-size-buffers=1 leaky=downstream ! nvvidconv ! video/x-raw, format=(string)BGRx ! videoconvert ! video/x-raw, format=(string)BGR ! appsink
```
Comment (blaskoa) [inline sf-docker/.env:28]: This means that the jetson `run-jetson.sh` script won't work out-of-the-box. We could consider splitting this configuration to two separate variables e.g. `GstPipelineTemplate_linux` and `GstPipelineTemplate_jetson` and then using them like this:

in `docker-compose.yml`
```
- GstPipelineTemplate=${GstPipelineTemplate_linux}
```

in `jetson-docker-compose.yml`
```
- GstPipelineTemplate=${GstPipelineTemplate_jetson}
```

**Comment by blaskoa** [comment]
revise if `master` branch is the correct target - maybe it should target `dev`? maybe some release branch?

File: sf-docker/jetson-docker-compose.yml [hunk 3]
```diff
@@ -65,7 +65,7 @@ services:
       - AppSettings__USE_JAEGER_APP_SETTINGS
       - JAEGER_AGENT_HOST
       - S3Bucket__Endpoint
-      - GstPipelineTemplate
+      - GstPipelineTemplate_Jetson
```
Comment (blaskoa) [inline sf-docker/jetson-docker-compose.yml:68]: `- GstPipelineTemplate=${GstPipelineTemplate_Jetson}` needs to be here for it to work

**Review by blaskoa — APPROVED** [review]

**Pull request merged**